src/common/login.ts

```diff
@@ -34,7 +34,7 @@ const Login = (() => {
             : `${oauth_url}/oauth2/authorize?app_id=${deriv_app_id}&l=${language}&brand=${brand_name.toLowerCase()}${affiliate_token_link}${cookies_link}&platform=${sub_url}`
     }
 
-    const initOneAll = (provider) => {
+    const initOneAll = (provider: string): void => {
         const social_login_url = `${loginUrl()}&social_signup=${provider}`
```

Comment:
```suggestion
type TSocialProvider = "google" | "facebook" | "apple" 
const initOneAll = (provider: TSocialProvider) => {}
```
As far as I could find in our codebase these are the only possible values, let me know what you think about this ,thanks

Reply:
Thanks a lot. I'll check other functions to follow this approach 🙇